pkg/handlers/aws/mutation/metapatch_handler_test.go

```diff
@@ -6,12 +6,10 @@ package mutation
 import (
 	"testing"
 
-	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 
 	"github.com/d2iq-labs/capi-runtime-extensions/api/v1alpha1"
 	"github.com/d2iq-labs/capi-runtime-extensions/common/pkg/capi/clustertopology/handlers/mutation"
-	"github.com/d2iq-labs/capi-runtime-extensions/common/pkg/k8s/client/fakessa"
```

Comment:
So its ok to still use `"sigs.k8s.io/controller-runtime/pkg/client/fake"` because those did not rely on SSA

Reply:
Yes.